package.json

```diff
@@ -45,11 +46,16 @@
         "proof-of-membership",
         "monorepo"
     ],
-    "workspaces": [
-        "apps/*",
-        "packages/*",
-        "packages/contracts/contracts"
-    ],
+    "workspaces": {
+        "packages": [
+            "packages/*",
+            "apps/*",
+            "packages/contracts/contracts"
+        ],
+        "nohoist": [
+            "**/semaphore-subgraph/**"
+        ]
```

Comment:
Why is it necessary to add this section? It looks like the changes in the script work well and solve the issue.

Reply:
Hi @vplasencia, it's not necessary to use nohoist - but it is kind of best practice to use it, I think because it does the dependency Isolation and avoids unnecessary conflicts later. Let me know if we can remove it also - not needed.

Reply:
Hey @0xshikhar! Thank you for the reply. I think it's better to remove.

Reply:
Hey @vplasencia , just removed it. Thanks.

Reply:
Hey @vplasencia, let me know once you checked it